improver_tests/acceptance/test_checksums.py

```diff
@@ -58,8 +58,6 @@ def test_kgo_checksums():
         for path in data_paths:
             acc.verify_checksum(path)
     except Exception as e:
-        if acc.kgo_recreate():
-            recreate_checksum_file(data_paths)
```

Comment:
I think removing this call to `recreate_checksum_file` makes that function dead code without any other references.
Suggest removing the function definition (starting line 66) so that there's only one way to generate the checksums (the new one in `bin/improver-tests`) and no dead code.

Reply:
Done.